src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java

```diff
@@ -53,53 +48,9 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
             errorResponse.addError("Before running this event audio and video evidence must be uploaded");
             return errorResponse;
         }
-        final UserDetails userDetails = idamService.getUserDetails(userAuthorisation);
-        final boolean hasJudgeRole = userDetails.hasRole(JUDGE);
-        final boolean hasTcwRole = userDetails.hasRole(TCW);
-        final boolean hasSuperUserRole = userDetails.hasRole(SUPER_USER);
-        setSelectedAudioVideoEvidence(sscsCaseData);
-        setProcessAudioVideoActionDropdown(sscsCaseData, hasJudgeRole, hasTcwRole, hasSuperUserRole);
+        ProcessAudioVideoActionHelper.setSelectedAudioVideoEvidence(sscsCaseData);
```

Comment:
Don't you still need to call the `populateListItems` method from the helper class here?

Reply:
No, it's not used on the first page, only second and now gets populated by the mid event instead. There was also a bug that if you went back to the first page it was losing the evidence dropdown which is also fixed.